src/scrabble.js

```diff
@@ -1,5 +1,150 @@
-function scrabble() {
-  // write code here
+function checkIncorrect(letterArray) {
+  const allowedChars = [
+    'A',
+    'B',
+    'C',
+    'D',
+    'E',
+    'F',
+    'G',
+    'H',
+    'I',
+    'J',
+    'K',
+    'L',
+    'M',
+    'N',
+    'O',
+    'P',
+    'Q',
+    'R',
+    'S',
+    'T',
+    'U',
+    'V',
+    'W',
+    'X',
+    'Y',
+    'Z',
+    '{',
+    '}',
+    '[',
+    ']'
+  ]
+  for (let i = 0; i < letterArray.length; i++) {
+    if (!allowedChars.includes(letterArray[i])) {
+      return 0
+    }
+  }
+  if (letterArray.includes('[') && !letterArray.includes(']')) {
+    return 0
+  }
+
+  if (letterArray.includes('{') && !letterArray.includes('}')) {
+    return 0
+  }
+
+  if (!letterArray.includes('[') && letterArray.includes(']')) {
+    return 0
+  }
+
+  if (!letterArray.includes('{') && letterArray.includes('}')) {
+    return 0
+  }
+}
+
+function specialCases(letterArray) {
```

Comment:
again, bad function name, special cases doesn't tell me what this function is meant to do